src/simulators/matrix_product_state/matrix_product_state_tensor.hpp

```diff
@@ -18,6 +18,7 @@
 
 #include <cstdio>
 #include <iostream>
+#include <iomanip>
```

Comment:
Why this? :)

Reply:
The standalone doesn't compile now without this. Before, I guess, it ended up getting iomanip through some dependency